tests/e2e/conftest.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (C) 2024 Red Hat, Inc.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+import json
+import os
+from pathlib import Path
+
+import pytest
+import simple_logger.logger
+from jinja2 import Environment
+from jinja2 import select_autoescape
+from jinja2.loaders import FileSystemLoader
+
+from cli.objects.jira_base import Jira
+
+logger = simple_logger.logger.get_logger(__name__)
+
+
+@pytest.fixture
+def job_dir(build_id):
+    yield Path("/tmp").joinpath(build_id)
```

Comment:
please use pytest tmp dir

Reply:
This test invokes the report command as if it were being run from the CLI. I do not think there is currently a way to change the job directory this way. This could be patched, but I would like to make sure that the report command is invoked at least once without any patching involved.